assignment10/challenges.txt

```diff
@@ -0,0 +1,10 @@
+My main challenges were:
+
+1. Accessing the list of vulnerabilities: The <ul> with the top 10 risks did not have an id or class, 
+so I had to use the <h2> element before it and XPath to locate the list.
```

Comment:
Good! This is a great case where xpath is helpful: I think most of us want to use css selectors but it won't always scale well.